special-pages/pages/new-tab/app/components/ShowHide.module.css

```diff
@@ -18,23 +18,31 @@
         width: 2rem;
         border-radius: 50%;
         padding-inline: 0;
+        background-color: transparent;
+        color: var(--ntp-text-muted);
 
-        svg rect {
+        .iconBlock,
+        svg path {
+            background-color: transparent;
+            box-shadow: none;
             fill-opacity: 0;
+            transition: all 0.3s ease-in;
         }
 
         &:hover {
-            transition: all 0.3s ease-in;
-            svg rect {
-                fill: black;
-                fill-opacity: 0.06;
+            .iconBlock {
+                backdrop-filter: blur(48px);
+                background-color: var(--color-black-at-6);
 
                 [data-theme=dark] & {
-                    fill: white;
-                    fill-opacity: 0.12;
+                    background-color: var(--color-white-at-12);
                 }
             }
-        }
+
+            svg path {
+                fill-opacity: 1;
```

Comment:
Because this is only on hover now, it means the privacy stats widget is now hiding it's expand/collapse button. I think we can solve this with composition of classes. 

For example, let's have a class that opts-in to auto-hiding, or the opposite. But either way I think we should have a dismiss button that 'just works' - and any additional hiding/showing can be opt-in by applying additional classes or data attributes.

<img width="625" alt="image" src="https://github.com/user-attachments/assets/5fb2f5ec-fce1-444b-a9cf-2d79c893e62e" />

Reply:
lol this came about b/c of the switch to the `[data-theme]` update, it managed to shuffle the specificity. it has been addressed.